lib/swaps/Swaps.ts

```diff
@@ -562,27 +562,29 @@ class Swaps extends EventEmitter {
     }
 
     if (height) {
-      this.logger.debug(`got block height of ${height} for ${takerCurrency}`);
+      this.logger.debug(`got ${takerCurrency} block height of ${height}`);
 
       const routeAbsoluteTimeLock = makerToTakerRoutes[0].getTotalTimeLock();
-      this.logger.debug(`choosing a route with total time lock of ${routeAbsoluteTimeLock}`);
       const routeTimeLock = routeAbsoluteTimeLock - height;
```

Comment:
Didn't we settle on the term `lockDuration`? At least the user-facing log output should say that.